pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -174,7 +175,9 @@ func (p *PolicyData) EnsureRBACPolicy() genericapiserver.PostStartHookFunc {
 		// initializing roles is really important.  On some e2e runs, we've seen cases where etcd is down when the server
 		// starts, the roles don't initialize, and nothing works.
 		err := wait.Poll(1*time.Second, 30*time.Second, func() (done bool, err error) {
-			client, err := clientset.NewForConfig(hookContext.LoopbackClientConfig)
+			adminLogicalClusterConfig := clientgorest.CopyConfig(hookContext.LoopbackClientConfig)
+			adminLogicalClusterConfig.Host += "/clusters/admin"
```

Comment:
TODO: we should not be hard-coding this as the baseline default, there is a const and builders for that

Reply:
You're right @stevekuznetsov . Just didn't take the time to extract the const from where it was used. Sorry if it brought problems to you :-(